lib/modules/story_editor/widgets/actor_picker_sheet.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:andax/models/actor.dart';
+import 'package:andax/modules/story_editor/screens/story_editor.dart';
+import 'package:andax/modules/story_editor/widgets/actor_tile.dart';
+import 'package:andax/shared/widgets/modal_picker.dart';
+import 'package:andax/shared/widgets/rounded_back_button.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'actor_editor_dialog.dart';
+
+void showActorPickerSheet(
+  BuildContext context,
+  ValueSetter<Actor?> onSelect, [
+  String? selectedId,
+]) {
+  final editor = context.read<StoryEditorState>();
+  showModalPicker<Actor>(
+    context,
+    (context, scroll) {
+      final actors = editor.story.actors.values.toList();
+      return Provider.value(
+        value: editor,
+        child: Builder(
```

Comment:
I don't understand why we need `Provider.value` and `Builder` inside it. I'm leaving this comment as a reminder for myself to check the docs (I wanted to complete the review quickly as I was busy :P)

Reply:
Provider only keeps its values within the current route. When you open a dialog, sheet, or screen, a new route is created where the provided values don't propagate.